### PR TITLE
Reset password

### PR DESCRIPTION
**Pull request by jburel**
To test:
 * Log as an admin
 * Go to the admin tab
 * Select a user and change password. 
 * No error message should be displayed.

### CODE REVIEW

File: components/insight/SRC/org/openmicroscopy/shoola/env/data/AdminServiceImpl.java [hunk 1]
```diff
@@ -623,6 +620,7 @@ public GroupData reloadGroup(SecurityContext ctx, GroupData group)
 				//check that the user is not ldap
 				String ldap = gateway.lookupLdapAuthExperimenter(ctx,
 						exp.getId());
+				System.err.println(ldap);
```
Comment (joshmoore) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/AdminServiceImpl.java]: System.err.println heads up.

Comment (jburel) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/AdminServiceImpl.java]: crap...
thanks

**Comment by mtbc** [comment]
Copyright. (-:

**Comment by mtbc** [comment]
This PR does seem to be working just fine and fixes http://trac.openmicroscopy.org.uk/ome/ticket/12054. The commits since the last merge look good. Would some confirmation dialog (such as `EditorComponent.passwordChanged`) or some other confirmation of success be a good addition?

**Comment by mtbc** [comment]
@rleigh-dundee: I hope this improves the anchor names suitably.

**Comment by mtbc** [comment]
Great, thank you. Good to merge.

**Comment by jburel** [comment]
--no-rebase

**Pull request merged**